examples/ConfigurationStoreBackup/KeyLabel.cs

```diff
@@ -10,17 +10,7 @@ internal struct KeyLabel
         public KeyLabel(string key, string label)
         {
             Key = key;
-            Label = label;
-        }
-
-        public override bool Equals(Object obj)
-        {
-            if (obj is KeyLabel keyLabel)
-            {
-                return string.Equals(this.Key, keyLabel.Key, StringComparison.Ordinal)
-                    && string.Equals(this.Label, keyLabel.Label, StringComparison.Ordinal);
-            }
-            return false;
+            Label = label ?? string.Empty;
```

Comment:
We need an equality overload because we need to match both key and label. Key+Label is the unique identifier for any key-value in AppConfig. We can change it to case-insensitive comparison though.

Reply:
> We can change it to case-insensitive comparison though.

Why is it?

Reply:
So I tried that but the override was completely useless when deployed as is to Azure. When a config key with no label came through it was not being matched. Removing the override and setting a default on a null string was what fixed it.

Reply:
@zhenlan You're right, its not case insensitive. I remembered incorrectly. 
@calloncampbell thanks for catching this! We can leave the override and just assign empty string if label is null in the KeyLabel constructor (like you already have here). That should fix the issue.